hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -237,13 +237,15 @@ public StateView(
 			}
 			var token = tokenStore.get(id);
 			var info = TokenInfo.newBuilder()
```

Comment:
1) We will have to add `supplyType` as-well
2) StateViewTest.java should be updated with checks for the new properties as-well